components/suggestion/modal_suggestion_list.jsx

```diff
@@ -49,15 +49,19 @@ export default class ModalSuggestionList extends React.PureComponent {
     componentDidMount() {
         if (this.container.current) {
             const modalBodyContainer = getClosestParent(this.container.current, '.modal-body');
-            modalBodyContainer.addEventListener('scroll', this.onModalScroll);
+            if (modalBodyContainer) {
+                modalBodyContainer.addEventListener('scroll', this.onModalScroll);
```

Comment:
Are the changes in this file necessary if we are not using `ModalSuggestionList` in the embedded forms? It looks like it is only used for int dialogs, by looking at `components/embedded_form/embedded_form.jsx` and `components/interactive_dialog/dialog_element/dialog_element.jsx`

Reply:
You are right, these changes may not be needed. Will remove.